Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM kbase/runtime:latest
+
+WORKDIR /opt/run/
+
+# set the GOPATH
+ENV GOPATH=/opt/go PATH=$PATH:$GOPATH/bin
+RUN mkdir /opt/go
+
+# install dependencies for SHOCK
+RUN \
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10 && \
+    echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | sudo tee /etc/apt/sources.list.d/mongodb.list && \
+    apt-get update && \
+    apt-get install -y \
+        libsasl2-dev libsasl2-modules-gssapi-mit \
+        mongodb-10gen=2.4.14 \
+        supervisor \
+        expect
+
+# update golang to 1.3.x
```

Comment:
KBase is running on an older version of Shock.  The develop branch of Shock requires 1.6, but it is unclear what that will break.  Keith and I discussed this briefly, and I was going to test with golang 1.6 to see if anything broke.  However, I won't be sure until the other components are integrated, so I was going to wait until then and try a newer version of golang.  Newer versions of Shock may be required for newer golang versions, and then that may bring API changes, etc.

Anyway, this matches current deployments of KBase.  Happy to test out newer versions of things as well.